### DISCUSSION TITLE
win10登录之后 音乐库打开卡住不动

### DISCUSSION THREAD

**Discussion by 9527945945** [qier222/YesPlayMusic#1126] [Q&A]
win10登录之后音乐库打开卡住不动 只能退出 非常的影响使用 希望尽快修复

**Comment by pan93412** [comment]
f5cdbea37993f313a53498d7572fdd5d7d6aac68 應已解決。請從 GitHub Actions 的 Artifact 處下載測試版。若沒問題，請幫我 Mark as answer，謝謝！
